integration_test.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"math/rand"
+	"os"
+	"os/exec"
+	"testing"
+	"text/template"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/api/iterator"
+)
+
+// integration test validates the end-to-end fluentbit and bigquery pipeline
+func TestPipeline(t *testing.T) {
+	logFilePath := "./logfile.log"
+	configFilePath := "./fluent-bit.conf"
+	ctx := context.Background()
+
+	// Set up BigQuery client
+	client, err := bigquery.NewClient(ctx, "bigquerytestdefault")
```

Comment:
Okay I can change this - however since Tanishqa and I don't have projects that we can actually run the integration test with, how would this work? Do we want to keep this file for our testing (but not in the final repo)

Reply:
What about the project "summer2024-raahi" and "summer2024-tpuhan"?

Reply:
Right - I was under the impression we didn't have the right authorization for that - but I just tried it and it seems to work fine. I will work on this change then.

Reply:
Done.